src/test/java/org/zaproxy/zap/extension/hud/ui/firefox/badsite/BadSiteUnitTest.java

```diff
@@ -167,30 +141,4 @@ private String getHudUtilsFilesUrl(HUD hud) throws MalformedURLException {
         }
         return utilsUrl;
     }
-
-    /**
-     * Obtain the HUD API URL. Target sites should not be able to do this, but we're testing defence
-     * in depth.
-     *
-     * @param driver
-     * @return
-     * @throws MalformedURLException
-     */
-    private String getHudApiUrl(HUD hud) throws MalformedURLException {
-
-        // Its worth noting that target sites wont be able to do this ;)
-        // We're just checking that even if the API URL is leaked somehow that the target site wont
-        // be able to use it
-        hud.getWebDriver().get(getHudUtilsFilesUrl(hud));
-        String utilsSrc = hud.getWebDriver().getPageSource();
-        int defnOffset = utilsSrc.indexOf("ZAP_HUD_API");
```

Comment:
There are still some references to `ZAP_HUD_API` in this test class that should be removed.

Reply:
Fixed